README.md

```diff
@@ -152,6 +152,10 @@ Note that for WebAssembly assets are downloaded on demand, as can be seen in the
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/UI/PackageResources)
 
+### Pet Adopt
+
```

Comment:
Can we add a description in here as to why this sample exists. Perhaps a link to the blog post

Reply:
Once the blog is published then we could update this description with the link